deploy/generate_ca.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
```

Comment:
the Kube builder documentation talks about using cert-manager for this sort of thing, can't we do the same?

I don't like assuming the openssl CLI is installed locally, That is not always the case.

Reply:
Installing cert-manager with configuring all the necessary stuff  such as Issuers, CertificateRequests etc to have just one cert generated is a huge overkill for me. I can hardly imagine an developer machine w/out having `openssl`.